webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/Table.js

```diff
@@ -160,34 +162,39 @@ export const Table = ({
               </Td>
             </Tr>
           )}
-          {children ||
-            results.map((result, rowIndex) => {
-              const rowActions = actions(result);
-              return (
-                <Tr key={rowIndex} ouiaId={`table-row-${rowIndex}`} isHoverable>
-                  {showCheckboxes && (
-                    <RowSelectTd
-                      rowData={result}
-                      selectOne={selectOne}
-                      isSelected={isSelected}
-                      idColumnName={idColumn}
-                    />
-                  )}
-                  {columnNamesKeys.map(k => (
-                    <Td key={k} dataLabel={keysToColumnNames[k]}>
-                      {columns[k].wrapper
-                        ? columns[k].wrapper(result)
-                        : result[k]}
+          {!childrenOutsideTbody &&
+            (children ||
```

Comment:
So if I pass in `children`, but at the same time set `childrenOutsideTbody` to `true`, the `children` won't be rendered?

Reply:
they will be, just not under the tbody
```js
 </Thead>
        {childrenOutsideTbody ? children : null}
<Tbody>
```

Reply:
Some days I'm just blind :)